tested/testsuite.py

```diff
@@ -289,9 +290,15 @@ def get_used_features(self) -> FeatureSet:
         return NOTHING
 
     def get_data_as_string(self, resources: Path) -> str:
-        file_path = _resolve_path(resources, self.expected_path)
-        with open(file_path, "r") as file:
-            return file.read()
+        file_content = []
+        for i in range(len(self.content)):
+            if self.content_type[i] == TextChannelType.FILE:
+                file_path = _resolve_path(resources, self.content[i])
+                with open(file_path, "r") as file:
+                    file_content.append(file.read())
+            else:
+                file_content.append(self.content[i])
+        return "\n".join(file_content)
```

Comment:
I would at at least some indication of file separation? Otherwise this will be very confusing